vision/unit_tests/bottleReader_unitTest.py

```diff
@@ -0,0 +1,50 @@
+"""
+Testing vision.deskew.coordinate_lengths.py
+"""
+
+import json
+import os.path
+from typing import TypedDict
+import unittest
```

Comment:
The `unittest` package isn't currently in the poetry environment for the SUAS2023 repo. Is it necessary for the test's functionality, or can the file run without the package?